### PR TITLE
EPMFARMATS-12587 ver. 2 

### PR DESCRIPTION
**Pull request by overdd**

### CODE REVIEW

File: EPMFARMATS-12587.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+//https://learn.javascript.ru/arrow-functions-basics
```
Comment (SiarheiRylach) [inline EPMFARMATS-12587.js:1]: encoding issue

File: EPMFARMATS-12587.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	else no();
+}
+
+ask( "�� ��������?",
+	yes => alert("�� �����������."),
+	no => alert("�� �������� ����������.")
+);
+
+//1. create a function of adding two numbers.
+const func = (a, b) => alert(a + b);
```
Comment (SiarheiRylach) [inline EPMFARMATS-12587.js:16]: return, not alert

File: EPMFARMATS-12587.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+ask( "�� ��������?",
+	yes => alert("�� �����������."),
+	no => alert("�� �������� ����������.")
+);
+
+//1. create a function of adding two numbers.
+const func = (a, b) => alert(a + b); 
+
+//2. create a  function of determining the name(if your name, then hello + name).
+const func = (name) => (name == "Anton" ? alert("Hello, " + name) : alert("I don't know you, " + name));
```
Comment (SiarheiRylach) [inline EPMFARMATS-12587.js:19]: the variable is already declared

Comment (SiarheiRylach) [inline EPMFARMATS-12587.js:19]: return, not alert

File: EPMFARMATS-12587.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+//3. create of calculating the type of argument and output to the console.
+const func = (element) => console.log(typeof(element));
+
+//4. create a function of selecting even elements of the array(returns a new array).
+function selectEvenElement(inputArray) {
+	const outputArray = [];
+	for (let element of inputArray) {
+		if (element % 2 == 0) outputArray.push(element);
```
Comment (SiarheiRylach) [inline EPMFARMATS-12587.js:28]: better to use strict comparison

**Comment by overdd** [comment]
encoding, return, strict comparison and file conflicts issues are fixed

**Comment by overdd** [comment]
func redeclaration issue fixed

**Comment by overdd** [comment]
const func = (a, b) => (a + b);
replaced with
let func = (a, b) => (a + b);

**Pull request merged**